semanticdb-javac/BUILD

```diff
@@ -0,0 +1,41 @@
+load("@bazel_skylib//rules:common_settings.bzl", "string_flag")
+load("@rules_java//java:defs.bzl", "java_import", "java_library", "java_plugin")
+
+package(
+    default_visibility = ["//visibility:public"],
+)
+
+config_setting(
+    name = "is_enabled",
+    flag_values = {":enabled": "true"},
+)
+
+string_flag(
+    name = "enabled",
+    values = ["true", "false"],
+    build_setting_default = "false",
+)
```

Comment:
What's the reason for having both `is_enabled` and `enabled`? 😅

Reply:
One of them defines the name when specifying the flag from the command-line  `bazel build //... //semanticdb-javac:enabled=true`.

The  `is_enabled` target is used as a filter for a `select(...)` clause.

See official docs https://bazel.build/docs/configurable-attributes